src/settings.rs

```diff
@@ -4,10 +4,30 @@
 //! in order to avoid redundancy.
 
 /// This application's name.
+#[allow(dead_code)]
 pub const APP_NAME: &str = "commit-analyzer";
 
 /// This application's tool tip.
 ///
 /// In order to show a brief usage synopsis to the user, this global variable
 /// sets the information about the required and optional command line arguments.
+#[allow(dead_code)]
 pub const APP_TOOL_TIP: &str = "<FILE> [OPTIONS]";
+
+/// This application's version information.
+///
+/// This constant represents this application's major version.
+#[allow(dead_code)]
+pub const VERSION_MAJOR: u8 = 0u8;
+
+/// This application's version information.
+///
+/// This constant represents this application's minor version.
+#[allow(dead_code)]
+pub const VERSION_MINOR: u8 = 1u8;
+
+/// This application's version information.
+///
+/// This constant represents this application's fix level.
+#[allow(dead_code)]
+pub const VERSIO_FIX_LEVEL: u8 = 0u8;
```

Comment:
Yes, there had been a typo.  But it is gone, now.

Sometimes, it is critical to know which version of an application you actually using, for instance with different versions of GCC or, to stay with Rust, the version of Rust itself.  This module, `crate::version`, was intended to be the storage for the version number.  One could also set up an automation to mirror changes in one place at the other one.  But I can also just remove it, if you prefer to not have a brief version information.  :wrench:

Reply:
Hmm, still not a fan of your handling of commits. 😅 

I'm against it for this small project. For a bigger project, where you expect regular updates and road maps, it makes more sense. But then I think it would be nice to actually parse the version number from the `Cargo.toml` file. But this is way out of scope for this..

Reply:
Complete removal or move to `usage` as hard coded strings? :wrench:

Reply:
I think complete removal. There are not really versions or differences between versions here.

Reply:
:wrench: :heavy_check_mark: